src/scss/constants.scss

```diff
@@ -275,7 +276,8 @@ $z-layers: (
   scrollbar-width: thin;
 
   // The webkit properties
-  @see https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar
+  @see https: //developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar
```

Comment:
Probably typo.

```suggestion
  // @see https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2278/commits/f423c3772cb4e9bb2a88f09ccc44a553786ca63d